vendor/forms.py

```diff
@@ -3,14 +3,16 @@
 from django import forms
 from django.apps import apps
 from django.conf import settings
+from django.contrib.sites.models import Site
+from django.db.models import TextChoices
```

Comment:
you probably don't need these imports since it seems like you moved the form to config.py

Reply:
maybe should move it back to be with the other forms for consistency?

Reply:
import removed :)